storybooknative/src/Button/index.js

```diff
@@ -1,13 +1,180 @@
+    },
+    [TOKENS.colorTextButtonBordered]: {
+      [TYPE_OPTIONS.PRIMARY]: theme.orbit.colorTextButtonPrimaryBordered,
+      [TYPE_OPTIONS.SECONDARY]: theme.orbit.colorTextButtonSecondaryBordered,
+      [TYPE_OPTIONS.INFO]: theme.orbit.colorTextButtonInfoBordered,
+      [TYPE_OPTIONS.SUCCESS]: theme.orbit.colorTextButtonSuccessBordered,
+      [TYPE_OPTIONS.WARNING]: theme.orbit.colorTextButtonWarningBordered,
+      [TYPE_OPTIONS.CRITICAL]: theme.orbit.colorTextButtonCriticalBordered,
+      [TYPE_OPTIONS.FACEBOOK]: theme.orbit.colorTextButtonFacebookBordered,
+      [TYPE_OPTIONS.GOOGLE]: theme.orbit.colorTextButtonGoogleBordered,
+    },
+  };
+  return tokens[name][type];
+};
+
+export const StyledButton = styled(View, props => {
+  const { bordered, theme, block, circled, disabled } = props;
+  const width = props.width ? `${props.width}px` : "auto";
+  return {
+    justifyContent: "center",
+    alignItems: "center",
+    height: getSizeToken(TOKENS.heightButton, props),
+    backgroundColor: bordered
+      ? theme.orbit.backgroundButtonBordered
+      : getTypeToken(TOKENS.backgroundButton, props),
+    width: block ? "100%" : width,
+    borderWidth: bordered ? "1px" : "0px",
+    borderColor: getTypeToken(TOKENS.borderColorButton, props),
+    borderRadius: circled
+      ? getSizeToken(TOKENS.heightButton, props)
+      : theme.orbit.borderRadiusNormal,
+    padding: 0,
+    paddingLeft: getSizeToken(TOKENS.paddingButton, props),
+    paddingRight: getSizeToken(TOKENS.paddingButton, props),
+    opacity: disabled && theme.orbit.opacityButtonDisabled,
+  };
+});
+
+StyledButton.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledText = styled(Text, props => {
+  const { bordered, theme } = props;
+  return {
+    color: bordered
+      ? getTypeToken(TOKENS.colorTextButtonBordered, props)
+      : getTypeToken(TOKENS.colorTextButton, props),
+    fontWeight: theme.orbit.fontWeightBold,
+    fontSize: getSizeToken(TOKENS.fontSizeButton, props),
+  };
+});
+
+StyledText.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledTouchableWrapper = styled(View, props => {
+  const width = props.width ? `${props.width}px` : "auto";
+  return {
+    overflow: "hidden",
+    width: props.block ? "100%" : width,
+    borderRadius: props.circled
+      ? getSizeToken(TOKENS.heightButton, props)
+      : props.theme.orbit.borderRadiusNormal,
+  };
+});
+
+StyledTouchableWrapper.defaultProps = {
+  theme: defaultTokens,
+};
+
+const Button = (props: Props) => {
+  const {
+    onPress = () => {},
+    children,
+    size = SIZE_OPTIONS.NORMAL,
+    type = TYPE_OPTIONS.PRIMARY,
+    ...rest
+  } = props;
+  let width;
+  if (props.block) {
+    width = "100%";
+  } else {
+    width = props.width ? `${props.width}px` : "auto";
+  }
+
+  if (Platform.OS === "android") {
+    return (
+      <StyledTouchableWrapper
+        size={size}
+        block={props.block}
+        width={props.width}
```

Comment:
Here you are using `props.width`, when in the component below we are using `width`. Could you fix that?